libbeat/_meta/fields.common.yml

```diff
@@ -25,3 +24,13 @@
           type: keyword
           description: >
             Error type.
+
+    - name: beat.name
+      type: alias
+      path: host.name
+      migration: true
+
+    - name: beat.hostname
+      type: alias
+      path: agent.hostname
```

Comment:
Why not `host.hostname`, for this one, BTW?

Reply:
It comes from the internal code where `beat.hostname` was changed to `agent.hostname`. The host.hostname is populate by the processor. In most cases these are identical but I would expect `host.hostname` could also be overwritten by some logs for example, agent not.